lib/common/socket/evloop.c.h

```diff
@@ -115,19 +115,97 @@ static void link_to_statechanged(struct st_h2o_evloop_socket_t *sock)
     }
 }
 
-static const char *on_read_core(int fd, h2o_buffer_t **input)
+#define NSEC_PER_SEC 1000000000
+
+static inline int64_t
+timespec_to_nsec(const struct timespec *a)
+{
+       return (int64_t)a->tv_sec * NSEC_PER_SEC + a->tv_nsec;
+}
+
+#if defined(__linux__) && !H2O_USE_LIBUV
```

Comment:
```suggestion
#if defined(__linux__)
```
This file is evloop.c.h, therefore the second condition is always met :-)

Reply:
ah right, copy paste bug 😄